### PR TITLE
Improve skipped, incomplete, risky, error, unkown and warning output

### PR DESCRIPTION
**Pull request by DanielBadura**

### CODE REVIEW

**Comment by DanielBadura** [comment]
@ventrec @ecrmnn what are you thinking about this? Could you give feedback if changes are needed?

File: src/PrettyPrinter.php [hunk 1]
```diff
@@ -52,7 +53,35 @@ public function endTest(Test $test, float $time): void
         }
```
Comment (navitronic) [inline src/PrettyPrinter.php:53]: The preceding four lines are no longer necessary

Comment (DanielBadura) [reply src/PrettyPrinter.php:53]: Thats true but i wont make any changes at the moment on this until one of the maintainers has given feedback on this in general. But thanks, i would also change some other things after looking at this code after such a time :D

**Comment by DanielBadura** [comment]
Ping @ventrec @ecrmnn

**Pull request merged**